### DISCUSSION TITLE
Suggestion: Start using semantic versions

### DISCUSSION THREAD

**Discussion by fredrike** [pypolestar/polestar_api#79] [General]
It seem a that versoning doesn't follow semantic versoning as the versions go from 1.3.9 - > 1.4.0.

Please have a look at https://semver.org/ an use that so one knows what type of changes have happened by looking at the version number. It is totally ok to have version 1.1.34 if there have not been any breaking changes but looking at the change log for https://github.com/leeyuentuen/polestar_api/releases/tag/1.3.4 it seem that this should have a  bump in `minor` at it states:

> note: some entities has been renames/removed

Really love the effort, keep up the good work!

**Comment by leeyuentuen** [comment]
i'll try to follow that rules on next release

**Comment by leeyuentuen** [comment]
i've done a different logic. 1.x was using to public release and the 1.1.x was using for beta patching that was not for public release.

**Comment by fredrike** [comment]
It is OK to start with semantic versoning although it's been something else before.
